flink-core/src/main/java/org/apache/flink/configuration/ConfigConstants.java

```diff
@@ -523,90 +523,50 @@
 
     // ------------------------ Mesos Configuration ------------------------
 
-    /**
-     * The initial number of Mesos tasks to allocate.
-     *
-     * @deprecated in favor of {@code MesosOptions#INITIAL_TASKS}.
-     */
+    /** @deprecated has no effect */
```

Comment:
Because they are still part of the public API, for better or worse. We could remove them but would need to add exclusions to the japicmp plugin, and I didn't wanna bother.